src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -119,91 +137,105 @@ export const PsvTechRecord: FormNode = {
+      validators: [{ name: ValidatorNames.Required }]
     },
     {
       name: 'euVehicleCategory',
       label: 'EU vehicle category',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.SELECT,
+      options: getOptionsFromEnum(EuVehicleCategories),
+      validators: [{ name: ValidatorNames.Required }]
     },
     {
       name: 'emissionsLimit',
       label: 'Emission limit (plate value)',
       value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }]
     },
     {
-      name: 'seatsLowerDeck',
-      label: 'Seats lower deck',
+      name: 'seatsUpperDeck',
+      label: 'Seats Upper deck',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }, { name: ValidatorNames.Required }],
+      id: 'style-float-right-small'
     },
     {
-      name: 'seatsUpperDeck',
-      label: 'Seats upper deck',
+      name: 'seatsLowerDeck',
+      label: 'Seats Lower deck',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 3 }, { name: ValidatorNames.Required }]
     },
     {
       name: 'standingCapacity',
       label: 'Standing capacity',
       value: '',
-
-      type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
-    },
-    {
-      name: 'vehicleSize',
-      label: 'Vehicle size',
-      value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 3 }, { name: ValidatorNames.Required }]
     },
     {
       name: 'numberOfSeatbelts',
       label: 'Number of seat belts',
       value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }, { name: ValidatorNames.Required }]
     },
+
     {
       name: 'seatbeltInstallationApprovalDate',
       label: 'Seatbelt installation approval date / type approved',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.DATE,
+      isoDate: false
+    },
+    {
+      name: 'vehicleSize',
+      label: 'Vehicle size',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      editType: FormNodeEditTypes.RADIO,
+      options: getOptionsFromEnum(VehicleSize)
```

Comment:
Very minor but could we capitalise `small` and `large` please
![image](https://user-images.githubusercontent.com/92087051/196948614-0898261a-1e45-40c6-9f74-6cd4c33d5a7a.png)

Reply:
Err I don't think so no, because the options come from an enum and the auto select is case sensitive

Reply:
You could use the `DefaultNullOrEmpty` pipe in the component on your options, which should capitalise it without affecting the underlying value. Or  you could pass in some options such as 
```js
   options: [
        { value: 'small', label: 'Small' },
        { value: 'large', label: 'Large' }
      ]
```
and not use the enum to achieve this

Reply:
Or change the `getOptionsFromEnum` function to capitalise the first letter of the `label` field

Reply:
"Or change the getOptionsFromEnum function to capitalise the first letter of the label field"

This is smart, I did this